### PR TITLE
Show Offer name on Admin Page Issue #218

### PR DESCRIPTION
**Pull request by k-granados**
Added Offer Name column with the associated offer names in the subscriptions page on the Publisher Solution.

### CODE REVIEW

File: src/SaaS.SDK.PublisherSolution/Controllers/HomeController.cs [hunk 1]
```diff
@@ -239,6 +239,7 @@ public IActionResult Subscriptions()
                         SubscriptionResultExtension subscriptionDetailExtension = this.subscriptionService.PrepareSubscriptionResponse(subscription);
                         Plans planDetail = this.planRepository.GetById(subscriptionDetailExtension.PlanId);
                         subscriptionDetailExtension.IsPerUserPlan = planDetail.IsPerUser.HasValue ? planDetail.IsPerUser.Value : false;
+                        subscriptionDetailExtension.OfferId = this.fulfillApiService.GetSubscriptionByIdAsync(subscription.AmpsubscriptionId).GetAwaiter().GetResult().OfferId;
```
Comment (santhoshb-msft) [inline src/SaaS.SDK.PublisherSolution/Controllers/HomeController.cs:242]: Sorry, I think we already have offerid in the db. just like planId we should get this from db

**Comment by santhoshb-msft** [comment]
@ms-mgrant looks like the actual bacpac file is missing with this change.

**Comment by code4clouds** [comment]
@k-granados  I would recommend adding screenshots of the changes is included. It just makes easy to understand the impact of the changes. ;-)

**Comment by santhoshb-msft** [comment]
@ms-mgrant could you please get the latest backpac and sql file and make the changes please. There might be some latest changes missing.

**Comment by ms-mgrant** [comment]
> 

@k-granados Try resolving the merge conflicts in the kg-experiment branch: The incoming files will contain the updated script and bacpac.

**Pull request closed without merging**